### ISSUE TITLE
iOS ContextMenu only opens on long press, not tap

### ISSUE THREAD

**Issue by ckrutsinger** [jerry08/Plugin.ContextMenuContainer#3]
Context menu is opening with tap on Andorid, but requires a long press on iOS.

```
            <cm:ContextMenuContainer x:Name="ActionsInline" >
                <cm:ContextMenuContainer.MenuItems>
                    <cm:ContextMenuItem          
                        Text="Option 1"
                         />
                    <cm:ContextMenuItem 
                        Text="Option 2"
                        />
                </cm:ContextMenuContainer.MenuItems>
                <cm:ContextMenuContainer.Content  >
                    <Button Text="Tap here to choose" 
                           FontSize="Medium"
                           Margin="10,3,3,3"
                           HorizontalOptions="Start"
                           />
                </cm:ContextMenuContainer.Content>
            </cm:ContextMenuContainer>
```

v1.0.6

Expected behavior is to open on tap.

Platform affected: iOS only, Android works fine

Demo app: [https://github.com/ckrutsinger/MauiContextMenuPlugin.git](https://github.com/ckrutsinger/MauiContextMenuPlugin.git)

**Comment by jerry08** [comment]
I had tested with `ImageButton` and not `Button` content. I just found that .NET MAUI `ImageButton` native element on iOS is `UIButton` whereas the native element for MAUI `Button` is a `UIView`. So I had to get the `UIButton` from the subviews in the `UIView`. I updated the package so the issue is fixed now.

Also, I didn't include this in the readme yet but I added 2 options in `ContextMenuContainer` called `ShowOnClick` (the default value is `true`) and `LongPressDuration` (applies to Android only and the default value is 500 milliseconds).
